### DISCUSSION TITLE
Ping impossible when few hours apppend

### DISCUSSION THREAD

**Discussion by fromnowhereuser** [angristan/wireguard-install#410] [Help]
I have 2 RPI that act has client, and one linux server.

I cant ping the server from the PI. But, i want to ping the Pi from the server, it fails.
And IF i ping again from the PI, the server can ping the Pi again.

It feels like the PI ping the server, "it wakes"up the VPN

**Comment by CesarPetrescu** [comment]
Read this: https://www.reddit.com/r/WireGuard/comments/i8ty2v/keep_alive_needed/
Basically you need to add PersistentKeepalive = x 
where x is the interval in seconds (i have it on 25 on my linux boxes)

**Comment by crazygit** [comment]
Can this script auto generate `PersistentKeepalive = x` in client config file ?
